src/portable/nordic/nrf5x/dcd_nrf5x.c

```diff
@@ -453,9 +453,11 @@ bool dcd_edpt_xfer (uint8_t rhport, uint8_t ep_addr, uint8_t * buffer, uint16_t
 
   xfer_td_t* xfer = get_td(epnum, dir);
 
+  dcd_int_disable(rhport);
```

Comment:
oh, I think I did put the int disable/enable here before, but somehow got reverted.